plugin/action/rename/config.go

```diff
@@ -0,0 +1,80 @@
+package rename
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+
+	insaneJSON "github.com/vitkovskii/insane-json"
+)
+
+type Config []string
+
+const (
+	NotFoundIdx = -1
+)
+
+var _ json.Unmarshaler = (*Config)(nil)
+
+func (c *Config) Find(key string) (string, int) {
+	keyValues := *c
+	for i := 0; i < len(keyValues); i += 2 {
+		if keyValues[i] == key {
+			return keyValues[i+1], i
+		}
+	}
+	return "", NotFoundIdx
+}
+
+func (c *Config) Remove(key string) {
+	_, idx := c.Find(key)
+	if idx == NotFoundIdx {
+		return
+	}
+
+	if idx == len(*c) {
```

Comment:
This condition seems to be always `false` since idx cannot be equal to `len(*c)`